ngraph/core/include/ngraph/pass/manager.hpp

```diff
@@ -102,6 +102,7 @@ class NGRAPH_API ngraph::pass::Manager
     /// \brief Set external PassConfig object.
     void set_pass_config(const std::shared_ptr<PassConfig>& pass_config)
     {
+        pass_config->add_disabled_passes(*m_pass_config);
```

Comment:
From my point of view it is unexpected that we disabled some passes in the constant argument in method which has a name `set_pass_config`

Reply:
During offline discussion decided to pass shared pass config to Manager constructor and remove set_pass_config method for pass::Manager.